lib/Driver/Compilation.cpp

```diff
@@ -1705,7 +1705,7 @@ static void writeIndexUnitOutputPathsToFilelist(llvm::raw_fd_ostream &out,
   for (auto &output : outputInfo.getIndexUnitOutputFilenames())
     out << output << "\n";
 }
-static void writeSupplementarOutputToFilelist(llvm::raw_fd_ostream &out,
+static void writeSupplementaryOutputToFilelist(llvm::raw_fd_ostream &out,
                                               const Job *job) {
```

Comment:
Nit: Please re-run clang format to re-align this

Reply:
A naive run of `clang-format version 14.0.1` (from `brew install clang-format`) results in this file being dramatically rewritten.

I'm happy to fix the indentation of the line below, but I'm uncomfortable rewriting the entire file (or risking rewriting the entire repository).

Reply:
The recommended workflow is to use `git-clang-format <branch base SHA>` to just reformat the lines touched on the branch. A bit more documentation here: https://github.com/apple/swift/blob/main/docs/HowToGuides/FAQ.md#how-do-i-format-my-changes